src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -533,7 +564,7 @@ private void showMore(boolean animate) {
 
 	private void showLess(boolean animate) {
 		showingMore = false;
-		ui.via.setLocation(null);
+		ui.via.setLocation((Location)null);
```

Comment:
No problem. I will revert my last commit except for the annotations.

The problem is that `setLocation(null)` is ambiguous. It can be both `setLocation(WrapLocation)` or `setLocation(Location)` since they both accept a single argument. Passing `null` to `setLocation(WrapLocation)`, however, does not make any sense. (This method can't even handle null, I should probably fix this.)